src/test/java/bowling/ui/OutputCuiTest.java

```diff
@@ -0,0 +1,62 @@
+package bowling.ui;
+
+import bowling.domain.Game;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class OutputCuiTest {
+
+    private Game game;
+
+    @BeforeEach
+    void setUp() {
+        game = new Game("tes");
+        game.throwBall(10); // 1
+        game.throwBall(10);
+        game.throwBall(10);
+        game.throwBall(10);
+        game.throwBall(10); // 5
+        game.throwBall(10);
+        game.throwBall(10);
+        game.throwBall(10);
+        game.throwBall(10); // 9
+    }
+
+    @DisplayName("X|XX")
```

Comment:
마지막 프레임은 최대 3번까지 투구가 가능하기 때문에, 3번을 투구한 경우 점수의 구분자가 더 필요해 보입니다.
View에서 이 부분을 처리해보시면 좋을 것 같습니다.
```suggestion
    @DisplayName("X|X|X")
```